example/cmd/microctl/waitready.go

```diff
@@ -18,7 +21,7 @@ func (c *cmdWaitready) Command() *cobra.Command {
 		RunE:  c.Run,
 	}
 
-	cmd.Flags().IntVarP(&c.flagTimeout, "timeout", "t", 0, "Number of seconds to wait before giving up"+"``")
+	cmd.Flags().DurationVarP(&c.flagTimeout, "timeout", "t", 0, "Number of seconds to wait before giving up"+"``")
```

Comment:
Changing this to a duration means it is no longer a number of seconds? Presumably the user needs to specify something like "10s" instead? I'd leave it as an int and multiply `c.flatTimeout` by `time.Second` below.